### DISCUSSION TITLE
Need help to understand the instruction given in the chapter on Calibration

### DISCUSSION THREAD

**Discussion by nsengupta** [japaric/discovery#554] [Q&A] [answered]
I am a newbie in the Rust Embedded world and I am taking first steps using BBC Microbit V2.

From the fantastic book **https://docs.rust-embedded.org**, chapter [09](https://docs.rust-embedded.org/discovery/microbit/09-led-compass/calibration.html), on LED Compass: 

> If you do not want to play the game every time you restart your application during development feel free to modify the `src/main.rs` template to just use the same static calibration once you got the first one.

Somehow, my old brain is failing to understand the _same static calibration once you got the first one_ part.

Could someone please push me in the intended direction here?

**Comment by eldruin** [comment, marked as answer]
It means that once you have done the calibration once, you can copy the values and just create a constant `calibration` variable with them instead of calling `let calibration = calc_calibration(...)`

**Reply by nsengupta** [reply]
Ah, OK! I was overthinking obviously: how to do it _programmatically_?

Source change is much easier because the hardware is the same and therefore, calibration needs no repetition.

Thank you!
